paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/FlowControllerTest.kt

```diff
@@ -48,9 +48,6 @@ internal class FlowControllerTest {
     ) = runFlowControllerTest(
         networkRule = networkRule,
         integrationType = integrationType,
-        paymentOptionCallback = { paymentOption ->
```

Comment:
were all these callbacks not actually being called before? I'm curious why these are just deleted instead of being replaced with an assert on the turbine

Reply:
Correct! They weren't being called! Makes these not only confusing but wrong!!!

Reply:
:face_with_spiral_eyes: